src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Functions/ActivityNames.cs

```diff
@@ -4,9 +4,17 @@ internal static class ActivityNames
 {
     public const string CopyCsvFiles = nameof(CopyCsvFilesFunction.CopyCsvFiles);
     public const string ImportMetadata = nameof(ImportMetadataFunction.ImportMetadata);
+
+    public const string CreateMappings = nameof(ProcessNextDataSetVersionFunction.CreateMappings);
+
     public const string HandleProcessingFailure = nameof(HandleProcessingFailureFunction.HandleProcessingFailure);
 
     public const string ImportData = nameof(ProcessInitialDataSetVersionFunction.ImportData);
     public const string WriteDataFiles = nameof(ProcessInitialDataSetVersionFunction.WriteDataFiles);
-    public const string CompleteProcessing = nameof(ProcessInitialDataSetVersionFunction.CompleteProcessing);
+
+    public const string CompleteInitialDataSetVersionProcessing = 
+        nameof(ProcessInitialDataSetVersionFunction.CompleteInitialDataSetVersionProcessing);
+
+    public const string CompleteNextDataSetVersionMappingProcessing = 
+        nameof(ProcessNextDataSetVersionFunction.CompleteNextDataSetVersionMappingProcessing);
```

Comment:
There was spacing before to divide the activities which are generic with the activities which relate to processing the initial version. The spacing seems ad-hoc now so I think it would be good to try and group them again.

Reply:
Updated to leave the generic ones at the top, then initial data set ones next, and then next data set version ones last if that's OK?